boards/riscv/renode_riscv32/doc/index.rst

```diff
@@ -0,0 +1,55 @@
+.. _renode-riscv32:
+
+Renode RISCV32
+##############
+
+Overview
+********
+
+The RISCV32 Renode board configuration is used to emulate the RISCV32 architecture, just like QEMU.
+Contrary to QEMU, the peripherals of this board can be configured easily by editing the
+``renode_riscv32.repl`` script.
+
+Programming and debugging
+*************************
+
+Building
+========
+
+Applications for the ``renode_riscv32`` board configuration can be built as usual
+(see :ref:`build_an_application`):
+
+.. zephyr-app-commands::
+   :board: renode_riscv32
+   :goals: build
+
+Flashing
+========
+
+While this board is emulated and you can't "flash" it, you can use this
+configuration to run basic Zephyr applications and kernel tests in the Renode
+emulated environment. For example, with the :zephyr:code-sample:`synchronization` sample:
```

Comment:
please, add a paragraph about __how__ you are using renode to run this platform

Reply:
I don't get it - it is similar to how `qemu_riscv32` works:
```
west build -b renode_riscv32 -p auto -t run samples/hello_world
```

https://docs.zephyrproject.org/latest/boards/riscv/qemu_riscv32/doc/index.html

Reply:
ok, I didn't realise we had a `renode_riscv32` board.

Reply:
no, this PR is adding a new `renode_riscv32` board, it's just that the usage is similar to its `qemu_riscv32` equivalent